### PR TITLE
[NUI] Relocate RenderEffect files

### PR DESCRIPTION
**Pull request by jmmhappy**
### Description of Change ###
We had offline discussion that Utility folder is inappropriate for RenderEffect.
I made a new folder for them.

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/RenderEffects/BackgroundBlurEffect.cs [hunk 1]
```diff
@@ -19,42 +19,6 @@
...
-        /// </summary>
-        /// <remarks>
-        /// Created RenderEffect is immutable.
-        /// </remarks>
-        /// <param name="blurRadius">The blur radius value. The unit is pixel for standard cases.</param>
-        /// <returns>Background blur effect with given blur radius.</returns>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public static RenderEffect CreateBackgroundBlurEffect(float blurRadius)
-        {
-            return new BackgroundBlurEffect((uint)Math.Round(blurRadius, 0));
```
[the remark this answers is not in the document]
Comment (jmmhappy) [reply src/Tizen.NUI/src/public/RenderEffects/BackgroundBlurEffect.cs]: Good. I removed it. Please review thank you.

**Review by dongsug-song — APPROVED** [review]
approved

File: src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs [hunk 2]
```diff
@@ -48,50 +48,9 @@ internal RenderEffect(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr,
         /// <param name="blurRadius">The blur radius value. The unit is pixel for standard cases.</param>
         /// <returns>Background blur effect with given blur radius.</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static RenderEffect CreateBackgroundBlurEffect(float blurRadius)
+        public static BaseHandle CreateBackgroundBlurEffect(uint blurRadius)
```
[the remark this answers is not in the document]
Comment (rabbitfor) [reply src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs:51]: After this PR, `RenderEffect` does not need to be a normal class. For now it only exist to provide `CreateBackgroundBlurEffect()` so `static class` fits better.

[the remark this answers is not in the document]
Comment (jmmhappy) [reply src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs:51]: Hi. In this patch, I removed inheritance relationship, and the entire class BackgroundBlurEffect.
Now RenderEffect in NUI is a mere interface class with only a single static method CreateBackgroundBlurEffect(radius). As we add more render effects(in future), it's going to contain only a bunch of static functions. So the class works totally different from Dali's RenderEffect, and there is no reason to return this interface class instance.
But.. it is true.. that `return new BaseHandle;` is not quite intuitive.. Not only that we don't have BackgroundBlurEffect class that inherits RenderEffect anymore, I also lost a name to call what it returns. Dali checks if its type is valid.

[the remark this answers is not in the document]
Comment (jmmhappy) [reply src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs:51]: I will try static class
That seems better
Thank you

[the remark this answers is not in the document]
Comment (rabbitfor) [reply src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs:45]: > But.. it is true.. that return new BaseHandle; is not quite intuitive.. Not only that we don't have BackgroundBlurEffect class that inherits RenderEffect anymore, I also lost a name to call what it returns.

I fully agree with you. I don't think `view.SetRenderEffect(BaseHandle)` is our best choice. If the bi-directional reffering is matter here, please consider just removing `BackgroundBlurEffect` only.

```C#
public class RenderEffect : BaseHandle
{
    private RenderEffect(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
    {
    }

    public static RenderEffect CreateBackgroundBlur(float radius) => new RenderEffect(Interop.BackgroundBlurEffect.New(), true);
}
```

[the remark this answers is not in the document]
Comment (jmmhappy) [reply src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs]: Could you check the recent version?
Thank you 😀

**Review by rabbitfor — APPROVED** [review]

File: src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        /// <summary>
+        /// Create a background blur effect
+        /// </summary>
+        /// <remarks>
+        /// Created RenderEffect is immutable.
+        /// </remarks>
+        /// <param name="blurRadius">The blur radius value. The unit is pixel for standard cases.</param>
+        /// <returns>Background blur effect with given blur radius.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static RenderEffect CreateBackgroundBlurEffect(uint blurRadius)
```
Comment (hinohie) [inline src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs:49]:
```suggestion
        public static RenderEffect CreateBackgroundBlurEffect(float blurRadius)
```

The type of this API was changed!

From #6179 comments https://github.com/Samsung/TizenFX/pull/6179#discussion_r1656571999 , we recommend to use blurRadius as float, to follow as other platforms parameter.

Comment (jmmhappy) [reply src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs]: Done

**Pull request merged**